openchallenges/service_stack.py

```diff
@@ -157,11 +158,19 @@ def __init__(
             protocol=elbv2.ApplicationProtocol.HTTP,
         )
 
+        health_check = elbv2.HealthCheck(path="/", interval=duration.minutes(1))
+        if "apex" in construct_id:
+            health_check = elbv2.HealthCheck(
+                path="/api/v1/challenges?pageNumber=1&pageSize=1",
```

Comment:
```suggestion
                path="/health",
```

The LB should only rely on the contract with the apex. The apex actually has a dedicated endpoints for health checks that I defined [here](https://github.com/Sage-Bionetworks/sage-monorepo/blob/2c52dde49f5b2e0990390d852d6e599eeb56ef60/apps/openchallenges/apex/templates/http.conf.template#L82-L86).

Reply:
ok, should we just turn off AWS health check for now to help debugging.  

BTW, AWS health check is how AWS determines when to re-deploy or deploy additional containers for a service.  I would recommend using the AWS health check over apex in production.

Reply:
Doesn't the AWS health check need us to specify a path?

Reply:
> ok, should we just turn off AWS health check for now to help debugging.

It's up to you.

Reply:
ok, i've made changes per your suggestions.  I've left the custom configurations in but have also configured it to disable health checks for now.